ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/HARouting.java

```diff
@@ -336,10 +336,14 @@ private static void forwardTo(
         .map(location -> Integer.toString(location.getPartition()))
         .collect(Collectors.joining(","));
     // Add skip forward flag to properties
-    final Map<String, Object> requestProperties = ImmutableMap.of(
-        KsqlRequestConfig.KSQL_REQUEST_QUERY_PULL_SKIP_FORWARDING, true,
-        KsqlRequestConfig.KSQL_REQUEST_INTERNAL_REQUEST, true,
-        KsqlRequestConfig.KSQL_REQUEST_QUERY_PULL_PARTITIONS, partitions);
+    final Map<String, Object> requestProperties = new HashMap<>();
```

Comment:
You can leave this as an `ImmutableMap` and use the `builder` to add an extra entry

Reply:
+1